src/guide/scaling-up/tooling.md

```diff
@@ -88,7 +88,7 @@ Main article: [Using Vue with TypeScript](/guide/typescript/overview).
 
 - [Volar](https://github.com/johnsoncodehk/volar) provides type checking for SFCs using `<script lang="ts">` blocks, including template expressions and cross-component props validation.
 
-- Use [`vue-tsc`](https://github.com/johnsoncodehk/volar/tree/master/packages/vue-tsc) for performing the same type checking from the command line, or for generating `d.ts` files for SFCs.
+- Use [`vue-tsc`](https://github.com/johnsoncodehk/vue-tsc) for performing the same type checking from the command line, or for generating `d.ts` files for SFCs.
```

Comment:
Thanks for the PR!

I think the URL you've suggested is for the old repo. I believe the current URL is:
```suggestion
- Use [`vue-tsc`](https://github.com/johnsoncodehk/volar/tree/master/vue-language-tools/vue-tsc) for performing the same type checking from the command line, or for generating `d.ts` files for SFCs.
```

Reply:
#1991